docs/ref/models/fields.txt

```diff
@@ -86,14 +86,26 @@ If a field has ``blank=False``, the field will be required.
 
 .. attribute:: Field.choices
 
-A :term:`sequence` consisting itself of iterables of exactly two items (e.g.
-``[(A, B), (A, B) ...]``) to use as choices for this field. If choices are
-given, they're enforced by :ref:`model validation <validating-objects>` and the
-default form widget will be a select box with these choices instead of the
-standard text field.
-
-The first element in each tuple is the actual value to be set on the model,
-and the second element is the human-readable name. For example::
+A dictionary or iterable in the format described below to use as choices for
+this field. If choices are given, they're enforced by
+:ref:`model validation <validating-objects>` and the default form widget will
+be a select box with these choices instead of the standard text field.
+
+The key element is the actual value to be set on the model, and the second
+element is the human readable name. For example::
+
+    YEAR_IN_SCHOOL_CHOICES = {
+        "FR": "Freshman",
+        "SO": "Sophomore",
+        "JR": "Junior",
+        "SR": "Senior",
+        "GR": "Graduate",
+    }
+
+You can also pass a :term:`sequence` consisting itself of iterables of exactly
+two items (e.g. ``[(A, B), (A, B) ...]``). The first element in each tuple is
```

Comment:
The example with `(A, B), (A, B)` could lead to some confusion, with potentially having the user thinking that `A` and `A` are the same. I would suggest something like:
```suggestion
two items (e.g. ``[(A1, B1), (A2, B2) ...]``). The first element in each tuple is
```